docs/content/faqs/key-features.md

```diff
@@ -0,0 +1,78 @@
+
+#### SpatialOS schema auto-generation
+
+In future releases, you will be able to save time on writing out classes and structs by annotating the C# classes and structs you use. The [schema](https://docs.improbable.io/reference/latest/shared/glossary#schema) will be automatically generated for you in a standardised format. 
+
+## Native workflows
+
+We know that game developers are deeply familiar with and experienced in using Unity and the Unity Editor, so we are building features within Unity itself as much as possible, allowing you to experience a Unity-native workflow. 
+
+#### MonoBehaviour workflow and callbacks
+
+You can easily inject components into MonoBehaviour and get method callbacks when values change. You can use this to run code in reaction to changes in component values, events fired or commands received. Alternatively, you can get the latest value on each frame, if that’s your preference. 
+
+#### Automatic state synchronization
+
+When you update ECS component values, those changes are automatically synchronized across the network. We recognise that in some instances, you’ll want to use custom update logic, so we’ve made it easy to override the default behavior.
+
+## High performance
+
+We want to the SpatialOS GDK for Unity’s performance to be as fast as possible. A more efficient integration means more resources available to both the client and server; your game uses less resources so you can develop more content.
+
+We are using low allocation APIs, optional GameObjects and the Unity ECS to accomplish this. 
+
+#### Near-zero allocation for component updates
+
+In our older SpatialOS **SDK** for Unity, each component update allocates memory in managed C#. This was problematic as its garbage collection caused loading spikes, manifesting as dropped frames. 
+
+The new SpatialOS **GDK** for Unity directly deserializes networked data into ECS data types (which are largely structs), creating extremely low allocations for networked operations. (This functionality is currently in R&D and is something we’re treating as high priority.)
+
+#### GameObjects are optional
+
+You only use GameObjects when you want to. You have the option to do computation on entity data without the overhead of MonoBehaviours. Built-in functionality, such as the spawning pipeline, makes use of this.   
+
+## Customizability
+
+#### Full source access
+
+At Improbable, we’ve always believed that everyone benefits from open development and that code gets better, faster, when it’s public. We’re developing the SpatialOS GDK for Unity in the open so that you can better understand it, and directly impact its development. We welcome [issue reports](https://github.com/spatialos/UnityGDK/issues) and [feature requests](https://github.com/spatialos/UnityGDK#give-us-feedback) right now, and when we enter alpha, we’ll being accepting pull requests directly from the community.
```

Comment:
`we’ll being accepting pull requests directly from the community.`that sentence sounds weird. `well start accepting ...`?

Reply:
Yes, it's a typo - good spot, eagle eyes.